front/src/modules/ui/modal/components/ConfirmationModal.tsx

```diff
@@ -82,7 +87,9 @@ export function ConfirmationModal({
           }}
           onEnter={onConfirmClick}
         >
-          <H1Title title={title} fontColor={H1TitleFontColor.Primary} />
+          <StyledCenteredTitle>
```

Comment:
Ok, I wonder if we want this centered style title to be part of the modal. Let's discuss that with design later